client/state/invites/actions.js

```diff
@@ -196,57 +199,63 @@ export function createAccount( userData, invite ) {
 }
 
 export function generateInviteLinks( siteId ) {
-	return ( dispatch ) => {
-		wpcom
-			.undocumented()
-			.site( siteId )
-			.generateInviteLinks()
-			.then( () => {
-				dispatch( requestSiteInvites( siteId ) );
-			} );
+	return async ( dispatch ) => {
+		await wpcom.req.post( {
+			path: `/sites/${ siteId }/invites/links/generate`,
+			apiNamespace: 'wpcom/v2',
+		} );
+		dispatch( requestSiteInvites( siteId ) );
 	};
 }
 
 export function disableInviteLinks( siteId ) {
-	return ( dispatch ) => {
-		wpcom
-			.undocumented()
-			.site( siteId )
-			.disableInviteLinks()
-			.then( () => {
-				dispatch( requestSiteInvites( siteId ) );
-			} );
+	return async ( dispatch ) => {
+		await wpcom.req.post( {
+			path: `/sites/${ siteId }/invites/links/disable`,
+			apiNamespace: 'wpcom/v2',
+		} );
+		dispatch( requestSiteInvites( siteId ) );
 	};
 }
 
 export function acceptInvite( invite ) {
 	return async ( dispatch ) => {
-		const data = await wpcom.undocumented().acceptInvite( invite );
 		try {
+			const data = await wpcom.req.get(
+				{
+					path: `/sites/${ invite.site.ID }/invites/${ invite.inviteKey }/accept`,
+					apiVersion: '1.2',
+				},
+				{
+					activate: invite.activationKey,
+					include_domain_only: true,
+				}
+			);
+
 			if ( invite.role !== 'follower' && invite.role !== 'viewer' ) {
 				dispatch( receiveSites( data.sites ) );
 				// @TODO: Replace with Redux user fetching once lib/user is fully reduxified
 				await user().fetch();
 			}
 
-			if ( ! get( invite, 'site.is_vip' ) ) {
+			if ( ! invite.site.is_vip ) {
```

Comment:
Any chance `site` will not exist?

Reply:
I removed the `get` usages after realizing that `wpcom.undocumented().acceptInvite( invite )` was always accessing `invite.site.ID` without any guards. And it has worked.

Reply:
Nice, thanks for checking!